controllers/storagecluster/reconcile.go

```diff
@@ -506,6 +506,13 @@ func (r *StorageClusterReconciler) reconcilePhases(
 		}
 	}
 
+	// ensure the ocs-operator-config cm exists & has the correct values
+	err := r.ensureOCSOperatorConfig(instance)
```

Comment:
I think it will be better to move this even up in the reconciliation cycle as we might be waiting for status of rook resources and rook wont be running so there wont be a status update for those resources.

Reply:
Moved